app/scenes/Settings/Details.tsx

```diff
@@ -141,7 +141,9 @@ function Details() {
             subdomain ? (
               <>
                 <Trans>Your knowledge base will be accessible at</Trans>{" "}
-                <strong>{subdomain}.getoutline.com</strong>
+                <strong>
+                  {subdomain}.{new URL(env.URL).host}
```

Comment:
it's a safe assumption that `env.URL` is a valid URL

Reply:
This is going to show a double subdomain though…? I think it was fine hardcoded as subdomains are a cloud-only feature.

Reply:
It's not. It's using the env URL which will not have the team subdomain (or any subdomain presumably?)